### PR TITLE
feat: create 'stats' route with test chart (install nivo charts modules)

### PR DESCRIPTION
**Pull request by mgsecure**
Basic line chart to test route & nivo charts

### CODE REVIEW

**Comment by mgsecure** [comment]
image

File: src/stats/Functions.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import useWindowSize from '../util/useWindowSize.jsx'
+
+export function cl(obj) {
```
[the remark this answers is not in the document]
Comment (mgsecure) [reply src/stats/Functions.jsx:3]: ok

File: src/stats/StatsPage.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    return (
+        <React.Fragment>
+            <div style={{
+                minWidth: '390px', maxWidth: 720, height: '100%',
+                padding: '24px', backgroundColor: '#000',
+                marginLeft: 'auto', marginRight: 'auto',
+                fontSize: '1.5rem'
+            }}>
+
+                <DataContext.Provider value={{chartHeight}}>
```
[the remark this answers is not in the document]
Comment (mgsecure) [reply src/stats/StatsPage.jsx:20]: just temporary, wnat to talk to you about all of the data points that can be generated from the sheet/json..

File: src/stats/photoStatsData.json [hunk 3]
```diff
@@ -0,0 +1,174 @@
+{
```
[the remark this answers is not in the document]
Comment (mgsecure) [reply src/stats/photoStatsData.json:1]: just temporary, wnat to talk to you about all of the data points that can be generated from the sheet/json..

File: src/stats/StatsPage.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (mgsecure) [reply src/stats/StatsPage.jsx:20]: Most of the charts use the chart height that changes on screen width. Seemed most straightforward to pass it with the Data Context. (Hard to recognize when there's only one chart)

File: src/stats/chartDefaults.js [hunk 4]
```diff
@@ -0,0 +1,94 @@
+export const beltColors =
```
[the remark this answers is not in the document]
Comment (mgsecure) [reply src/stats/chartDefaults.js:1]: Most charts use the same one, but a few (as best I remember) use slightly specific ones. I'll take a look to make sure they're not the same in two places.

File: src/stats/siteStatsData.json [hunk 5]
```diff
@@ -0,0 +1,240 @@
+{
```
[the remark this answers is not in the document]
Comment (mgsecure) [reply src/stats/siteStatsData.json:1]: Temporary.

File: src/stats/Functions.jsx [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+    console.log(obj)
+}
+
+export function windowWidths() {
```
[the remark this answers is not in the document]
Comment (mgsecure) [reply src/stats/Functions.jsx:7]: Need your guidance..

File: src/stats/StatsPage.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (NiXXeD) [reply src/stats/StatsPage.jsx:20]: That is not what that context is for though. Let's chat more on this.

**Pull request merged**